src/Features/CSharp/Portable/CodeLens/CSharpCodeLensMemberFinder.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeLens;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeLens;
+
+[ExportLanguageService(typeof(ICodeLensMemberFinder), LanguageNames.CSharp), Shared]
+internal class CSharpCodeLensMemberFinder : ICodeLensMemberFinder
```

Comment:
```suggestion
internal sealed class CSharpCodeLensMemberFinder : ICodeLensMemberFinder
```

Reply:
i'm surprised this doesn't descend from a base class (so this would work for VB as well).